extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/RemoveDeploymentTriggerDecorator.java

```diff
@@ -16,7 +15,7 @@ public void andThenVisit(DeploymentConfigSpecFluent<?> deploymentConfigSpec, Obj
     }
 
     @Override
-    public Class<? extends Decorator>[] after() {
```

Comment:
This should not be removed.

The purpose of `RemoveDeploymentTrigger` is to remove triggers either provided by the user or by the `ApplyDeploymentTrigger` decorator. By removing this condition it's possible that `RemoveDeploymentTrigger` runs before there is anything to remove.